src/dex-ui/hooks/hederaTokenService/useCreateNonFungibleToken.tsx

```diff
@@ -0,0 +1,30 @@
+import { HTSMutations } from "./types";
+import { TransactionResponse } from "@hashgraph/sdk";
+import { useMutation } from "react-query";
+import { DexService } from "../../services";
+import { useDexContext } from "../useDexContext";
+
+export interface UseCreateNFTParams {
+  name: string;
+  symbol: string;
+  maxSupply: number;
+  treasuryAccountId: string;
+  supplyKey: string;
+}
+
+export function useCreateNonFungibleTokenToken(handleCreateTokenSuccessful: () => void) {
```

Comment:
"Token" is used twice here - not sure if that was intended

Reply:
Done